doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -63,26 +65,31 @@ if this option is not specified.
 
 =item B<-rawin>
 
-This indicates that signature input data is raw data, which for most signature
-algorithms (but not EdDSA) needs to be hashed by some message digest algorithm.
-This option can only be used with B<-sign> and B<-verify>
-and is implied by the Ed25519 and Ed448 algorithms.
+This indicates that the signature or verification input data is raw data,
+which is not hashed by any message digest algorithm.
 Except with EdDSA,
 the user can specify a digest algorithm by using the B<-digest> option.
+For signature algorithms like RSA, DSA and ECDSA,
+the default digest algorithm is SHA-256. For SM2, it is SM3.
+
+This option can only be used with B<-sign> and B<-verify>.
+For EdDSA (the Ed25519 and Ed448 algorithms) it must be used,
+which is implied since OpenSSL 3.5.
 
-The B<-digest> option implies B<-rawin>.
+The B<-digest> option implies B<-rawin> since OpenSSL 3.5.
 
 =item B<-digest> I<algorithm>
 
 This option can only be used with B<-sign> and B<-verify>.
 It specifies the digest algorithm which is used to hash the input data before
 signing or verifying it with the input key. This option could be omitted if the
 signature algorithm does not require one (for instance, EdDSA). If this option
```

Comment:
```suggestion
signature algorithm does not preprocess the input through a pluggable hash function before signing (for instance, EdDSA). If this option
```

Reply:
I presume you meant:

> This option could be omitted if the signature algorithm does not **require preprocessing** the input through a
pluggable hash function before signing (for instance, EdDSA).

Tentatively changed this way.